keras_cv/models/resnet_v2.py

```diff
@@ -390,15 +390,45 @@ def as_backbone(self):
         The output of the backbone model will be a dict with int as key and tensor as
         value. The int key represent the level of the feature output.
 
+        Args:
+            min_level: optional int, the lowest level of feature to be included in the
+                output. Default to model's lowest feature level (based on the model structure).
```

Comment:
Not sure how to interpret this, given all models are upsampled by 2x on every stage, wouldn't the lowest level always be 1?

Reply:
Not necessarily. Most of the backbone actually start with 2 (where the actually stack of conv blocks starts).

Reply:
Makes sense, thanks